akka-http/src/main/scala/akka/http/scaladsl/client/RequestBuilding.scala

```diff
@@ -18,37 +18,39 @@ import HttpMethods._
 trait RequestBuilding extends TransformerPipelineSupport {
   type RequestTransformer = HttpRequest ⇒ HttpRequest
 
+  implicit val timeout = Timeout(1.second)
```

Comment:
@ktoso Btw, also this addition should have been reported as a binary incompatibility.